### PR TITLE
Run SRCDS as steam user

### PR DESCRIPTION
**Pull request by rbreslow**
## Overview

- Configure dependent images to run under `steam` (with [`USER`](https://docs.docker.com/engine/reference/#user) block)
- Refactor README
- Run [shfmt](https://github.com/mvdan/sh)

Connects https://github.com/rbreslow/docker-steamcmd/pull/3

## Testing Instructions

Build the image against the latest docker-steamcmd:

```bash
CI=1 ./scripts/cibuild
```

Start SRCDS and confirm the deprecation notice is gone:

```bash
$ docker run -ti \
    -p 27015:27015 \
    -p 27015:27015/udp \
    --rm quay.io/rbreslow/garrysmodds:slim
Auto detecting CPU
Using default binary: ./srcds_linux
Server will auto-restart if there is a crash.
Couldn't load shader dll: game_shader_generic_garrysmod_srv.soConVarRef mat_dxlevel doesn't point to an existing ConVar
Game_srv.so loaded for "Garry's Mod"
Setting breakpad minidump AppID = 4000
Initializing Steam libraries for Workshop..
...
```

### CODE REVIEW

**Comment by rbreslow** [comment]
Review is blocked by https://github.com/rbreslow/docker-steamcmd/pull/3

File: Dockerfile [hunk 1]
```diff
@@ -1,18 +1,22 @@
 FROM quay.io/rbreslow/steamcmd:slim
 
-MAINTAINER Rocky Breslow <breslowrocky@gmail.com>
+LABEL maintainer="Rocky Breslow <breslowrocky@gmail.com>"
+
+USER root
```
Comment (rbreslow) [inline Dockerfile:5]: We have to switch to the `root` user first to have permissions to manage apt.

File: README.md [hunk 2]
```diff
@@ -7,22 +7,18 @@ This repository contains a `Dockerfile` designed to support Garry's Mod Dedicate
...
 ```
 
 Then, run the server and expose ports:
 
 ```bash
-$ docker run -ti \
+docker run -ti \
     -p 27015:27015 \
     -p 27015:27015/udp \
-    -p 27020:27020/udp \
```
Comment (rbreslow) [inline README.md:19]: These ports aren't necessary for running SRCDS for demonstration purposes.

See: https://developer.valvesoftware.com/wiki/Source_Dedicated_Server#Connectivity

File: Dockerfile (hunk 1 above)
Comment (starcraft66) [reply Dockerfile:5]: yes

File: README.md (hunk 2 above)
Comment (starcraft66) [reply README.md:19]: :+1:

**Review by starcraft66 — APPROVED** [review]
LGTM.

**Pull request merged**